src/views/favorites_panel.dot

```diff
@@ -19,28 +19,43 @@
           <div class="favorite_panel__swipe_window">
             <div class="favorite_panel__swipe_flow--{{= this.favoritePois.length }}">
               {{~ this.favoritePois:poi:i }}
-                <div class="favorite_panel__swipe_element" {{= click(this.go, this, poi) }} id="{{= `favorite_item_${i}` }}">
+                <div class="favorite_panel__item__more_container">
+
+                  <div class="favorite_panel__swipe_element" {{= click(this.go, this, poi) }} id="{{= `favorite_item_${i}` }}">
+
+                    <div class="favorite_panel__item__more_button">
+                      <i class="icon-more-horizontal" {{= click(this.toggleMore, this, i) }}></i>
```

Comment:
There is some unexpected behavior when clicking on the edge of the (highlighted!) button.
The click action should be defined on the button div itself.